.github/workflows/release.yml

```
@@ -92,6 +94,7 @@ jobs:
         working-directory: examples/react/w3console
         env:
           VITE_W3UP_PROVIDER: did:web:nft.storage
+          VITE_W3UP_SERVICE_BRAND_NAME: 'NFT.storage'
```

I think typeset like this normally:

```suggestion
          VITE_W3UP_SERVICE_BRAND_NAME: 'NFT.Storage'
```